### PR TITLE
new release for Workday groups support

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/common/components/adapters/WorkdayIdStore.php [hunk 1]
```diff
@@ -121,6 +121,14 @@ public function getAllActiveUsers()
                 var_export($allActiveUsers, true)
             ), 1532982679);
         }
+
+        foreach ($allActiveUsers as $key => $user) {
+            $allActiveUsers[$key]['Groups'] = implode(",", array_merge(
+                explode(" ", $user["reap_CompanyId"] ?? ""),
+                explode(" ", $user["reap_tree"] ?? "")
```
Comment (forevermatt) [inline application/common/components/adapters/WorkdayIdStore.php]: What do `reap_CompanyId` and `reap_tree` refer to? If possible, it seems worth getting those fields renamed in the Workday API to more meaningfully reflect their purpose.

Comment (Schparky) [reply application/common/components/adapters/WorkdayIdStore.php]: Good question. They're cryptic to me also. @fillup do you think it's possible to have them renamed? and possibly even combine them?

Comment (fillup) [reply application/common/components/adapters/WorkdayIdStore.php]: well, they are both specifically for REAP, so the name is accurate there, although if another app wanted to use them someday they'd be awkward, but I'll ask and see if they can change them I guess.

File: application/common/components/adapters/WorkdayIdStore.php [hunk 2]
```diff
@@ -121,6 +121,18 @@ public function getAllActiveUsers()
                 var_export($allActiveUsers, true)
             ), 1532982679);
         }
+
+        foreach ($allActiveUsers as $key => $user) {
+            $companyIDs = str_replace(" ", ",", $user["company_ids"] ?? "");
+            $ouTree = str_replace(" ", ",", $user["ou_tree"] ?? "");
+            if (strlen($companyIDs) > 0 && strlen($ouTree) > 0) {
```
Comment (forevermatt) [inline application/common/components/adapters/WorkdayIdStore.php]: Nice approach, here! That's a clever way to only add the comma when it's needed. :-)

**Pull request merged**